rescan.go

```diff
@@ -0,0 +1,116 @@
+package dcrlibwallet
+
+import (
+	"math"
+	"time"
+
+	"github.com/decred/dcrwallet/errors"
+	w "github.com/decred/dcrwallet/wallet/v3"
+)
+
+func (mw *MultiWallet) RescanBlocks(walletID int) error {
+
+	wallet := mw.WalletWithID(walletID)
+	if wallet == nil {
+		return errors.E(ErrNotExist)
+	}
+
+	netBackend, err := wallet.internal.NetworkBackend()
+	if err != nil {
+		return errors.E(ErrNotConnected)
+	}
+
+	if mw.IsRescanning() || !mw.IsSynced() {
```

Comment:
`mw.IsRescanning()` prevents being able to perform simultaneous rescans on multiple wallets. Is this intended? put another way, can't we do a `wallet.RescanBlocks()` method instead of a `mw.RescanBlocks(walletID)` method?

Reply:
The app doesn’t perform simultaneously rescan.